app/views/components/config/MucrOptionsConfig.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.components.config
+
+import config.AppConfig
+import forms.ManageMucrChoice
+import javax.inject.Inject
+import play.api.mvc.Call
+
+class MucrOptionsConfig @Inject()(appConfig: AppConfig) extends BaseConfig(appConfig) {
+
+  def backUrl(manageMucrChoice: Option[ManageMucrChoice] = None): Call =
+    if (appConfig.ileQueryEnabled)
+      if (manageMucrChoice.isDefined)
+        controllers.consolidations.routes.ManageMucrController.displayPage()
+      else
+        controllers.routes.ChoiceController.displayChoiceForm()
+    else
+      controllers.routes.ChoiceController.displayChoiceForm()
```

Comment:
```
if(appConfig.ileQueryEnabled && manageMucrChoice.isDefined) 
controllers.consolidations.routes.ManageMucrController.displayPage()
else
controllers.routes.ChoiceController.displayChoiceForm()
```

Small propose to avoid nested ifs

Reply:
There are three outcomes here, not two.  Maybe some java-style brackets would help

Reply:
Yeah, but two of them are the same. Those two elses contains the same redirection

Reply:
True- didn't spot that.  I was more focused on keeping the "ileQuery feature" behaviour clearly separated from the "non ileQuey feature" (so it's simpler to remove).  But on reflection, you are right.